package.json

```diff
@@ -1,6 +1,9 @@
 {
     "name": "aether-roc-gui",
     "version": "0.0.0",
+    "engines": {
+        "node": "16.0.0"
```

Comment:
I think you could go to `16.14.0` at least as this is used [here](https://github.com/onosproject/aether-roc-gui/blob/62df98016df06bd0131faf1fa6195f39c686b785/build/aether-roc-gui/Dockerfile#L6).